core/src/main/kotlin/transformers/psi/DefaultPsiToDocumentationTranslator.kt

```diff
@@ -100,14 +111,15 @@ object DefaultPsiToDocumentationTranslator : PsiToDocumentationTranslator {
                     )
                 },
                 null,
-                getComment(psi)
+                getComment(psi),
+                visibility = mapOf(platformData to psi.getVisibility())
             )
         }
 
         private fun parseField(psi: PsiField, parent: DRI): Property {
             val dri = parent.copy(
                 callable = Callable(
-                    psi.name,
+                    psi.name ?: "",
```

Comment:
I think `psi.name` returns `String`, so the elvis is redundant

Reply:
Changed.